# Source tree: downloads/uboot-race-test-39cd @ 39cd993e5d
#
# The first hunk was temporary instrumentation for the controlled-delay run and
# was removed afterwards.  The second hunk is the retained Makefile fix.

--- a/Makefile
+++ b/Makefile
@@ -948,6 +948,7 @@ quiet_cmd_copy = COPY    $@
       cmd_copy = cp $< $@
 
 u-boot.dtb: dts/dt.dtb FORCE
+	@sleep 3
 	$(call cmd,copy)

--- a/Makefile
+++ b/Makefile
@@ -1056,7 +1057,7 @@ u-boot-dtb.img u-boot.img u-boot.kwb u-boot.pbl u-boot-ivt.img: \
 		$(if $(CONFIG_SPL_LOAD_FIT),u-boot-nodtb.bin dts/dt.dtb,u-boot.bin) FORCE
 	$(call if_changed,mkimage)
 
-u-boot.itb: u-boot-nodtb.bin dts/dt.dtb $(U_BOOT_ITS) FORCE
+u-boot.itb: u-boot-nodtb.bin u-boot.dtb $(U_BOOT_ITS) FORCE
 	$(call if_changed,mkfitimage)
 
 u-boot-spl.kwb: u-boot.img spl/u-boot-spl.bin FORCE
